modules/home-environment.nix

```
@@ -585,7 +586,7 @@ in
       if config.submoduleSupport.externalPackageInstall
       then
         ''
-          if [[ -e $HOME/.nix-profile/manifest.json ]] ; then
+          if [[ -e $HOME/.nix-profile/manifest.json ]] || [[ -e $HOME/.local/state/nix/profile/manifest.json ]] ; then
```

Can use `config.xdg.stateHome`?

```suggestion
          if [[ -e $HOME/.nix-profile/manifest.json || -e "${config.xdg.stateHome}/nix/profile/manifest.json" ]] ; then
```